src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+}
+
+input {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
```

Comment:
Avoid styling elements using tag names, this increases specificity. Get use to styling using classes, so you can move this to .search styles